utilities/conv_gefs2datm/conv_gefs2datm.fort.csh

```diff
@@ -0,0 +1,52 @@
+#! /bin/csh -x
+#-- hyun-chul.lee@noaa.gov
+
+module load NetCDF/4.5.0
```

Comment:
This module does not exist on Hera.

In order to load the netcdf module, intel has to be loaded first

`module load intel netcdf`